package/broom.py

```diff
@@ -0,0 +1,310 @@
+    p1_total = set()
+    p1_oros = set()
+    p1_sevens = set()
+
+    p2_total = set()
+    p2_sevens = set()
+    p2_oros = set()
+
+    for card_id, card in self.deck.cards().items():
+      if (card.owner == self.pl1.name):
+        p1_total.add(card_id)
+        if card.suit == 'O': p1_oros.add(card_id) 
+        if card.face == 7: p1_sevens.add(card_id) 
+      else:
+        p2_total.add(card_id)
+        if card.suit == 'O': p2_oros.add(card_id)
+        if card.face == 7: p2_sevens.add(card_id)
+
+      if card_id == '7O':
+        self.pl1.award_point() if card.owner == self.pl1.name else self.pl2.award_point()
+      
+    if len(p1_total) > len(p2_total):
+      self.pl1.award_point()
+    elif len(p2_total) > len(p1_total):
+      self.pl2.award_point()
+    
+    if len(p1_oros) > len(p2_oros):
+      self.pl1.award_point()
+    elif len(p2_oros) > len(p1_oros):
+      self.pl2.award_point()
+    
+    if len(p1_sevens) > len(p2_sevens):
+      self.pl1.award_point()
+    elif len(p2_sevens) > len(p1_sevens):
+      self.pl2.award_point()
+    print(f'Points:\tPL1\tPL2\nOros:\t[{len(p1_oros)}]\t[{len(p2_oros)}]\nSevens:\t[{len(p1_sevens)}]\t[{len(p2_sevens)}]\nCards:\t[{len(p1_total)}]\t[{len(p2_total)}]')
+
+  def play_first_round(self, first_player, second_player):
+    p1_cards, p2_cards ,table_cards = self.deal_start()
+    first_player.new_hand(p1_cards)
+    second_player.new_hand(p2_cards)
+    self.table_cards = table_cards
+    return self
+
+  def play_round(self, first_player, second_player):
+    p1_cards, p2_cards ,table_cards = self.deal_start()
+    first_player.new_hand(p1_cards)
+    second_player.new_hand(p2_cards)
+    self.table_cards = table_cards
+    
+    last_scored = ''
+    cards_left = len(self.deck.order())
+    while len(self.deck.order()) > 0:
+      if len(first_player.hand) == 0 and len(second_player.hand) == 0:
+        p1_cards, p2_cards = self.deal_hand()
+        first_player.new_hand(p1_cards)
+        second_player.new_hand(p2_cards)
+        cards_left = len(self.deck.order())
+
+
+      # hand per player
+      while (len(first_player.hand) + len(second_player.hand) > 0):
+        if (len(first_player.hand)):
+          playable = self.valid_plays(first_player,self.table_cards)
+          play = first_player.get_play(playable)
+          if self.apply_play(play,first_player): last_scored = first_player.name
+
+        if (len(second_player.hand)):
+          playable = self.valid_plays(second_player,self.table_cards)
+          play = second_player.get_play(playable)
+          if self.apply_play(play,second_player): last_scored = second_player.name
+
+    # award last_player_to_score remaining cards
+    [self.set_card_owner(card_id, last_scored) for card_id, card in self.deck.cards().items() if card.owner == '']
+    self.apply_score()
+
+  def print_score(self):
+        print("Player 1 score: {}\nPlayer 2 score: {}".format(self.pl1.score, self.pl2.score))
+    
+  def returnJSON(self):
```

Comment:
it should be noted that this isn't actually returning "JSON" - it is still a python dict at the return statement - just happens to be a that you've put class data into a json friendly dict


in order for it to trully "returnJSON" you could consider refactoring the "jsonfiy" call to wrap the statement during return, and removing it from the calling functions

Reply:
briefly looked into this. when transfering jsonify to class, then rest of response is problematic, aka the client session id.  can do in future issue